packages/layer-selector/src/index.js

```diff
@@ -0,0 +1,3 @@
+import LayerSelector from './LayerSelector';
```

Comment:
This file seems a bit silly. Maybe webpack requires it?

Reply:
yes webpack requires it